rfcs/0002-ckb/0002-ckb.md

```diff
@@ -138,14 +138,14 @@ We refer to programs that generate new states and create new cells as Generators
 
 Cells are the primary state units in CKB, within them users can include arbitrary states. A cell has the following fields:
 
-- `capacity` - Size limit of the cell. A cell's size is the total number of bytes of all fields contained in it.
+- `capacity` - Size limit of the cell. A cell's size is the total number of ["CK Shannons"](../0015-ckb-cryptoeconomics/0015-ckb-cryptoeconomics.md#native-tokens) of all fields contained in it.
 - `data` - State data stored in this cell. It could be empty, however the total bytes used by a cell (including data), must always be less than or equal to its capacity.
 - `type`: State verification script.
 - `lock`: Script that represents the ownership of the cell. Owners of cells can transfer cells to others.
 
 A cell is an immutable object, no one can modify it after creation. Every cell can only be used once, it cannot be used as input for two different transactions. Cell ‘updates’ mark previous cells as history and create new cells with the same capacity to replace them. By constructing and sending transactions, users provide new cells with new states in them and invalidate previous cells that store old states atomically. The set of all current (or live) cells represents the latest version of all common knowledge in CKB, and the set of history (or dead) cells represents all historical versions of common knowledge.
 
-CKB allows users to transfer a cell's capacity all at once, or transfer only a fraction of a cell's capacity, which would in turn lead to more cells being created (e.g., a cell with capacity=10 can become two cells with capacity=5).
+CKB allows users to transfer a cell's capacity all at once, or transfer only a fraction of a cell's capacity, which would in turn lead to more cells being created (e.g., a cell which capacity is 9 bytes can become four cells which capacity are 2.25 bytes).
```

Comment:
It is a bit weird to name 2.25 bytes

Reply:
Yep, I feel weird, too!
I just want to show that bytes are divisible.

How about `10 bytes = 2.5 bytes * 4`?
Or I just keep the example as original?

Reply:
Is `1 byte` the minimum unit for (storage) capacity?

Reply:
Yes, I think I should split the two usages of capacity to two paragraphs.
I should not mix them, let them be a bit of a mess.

Reply:
I prefer not to introduce `Shannon` to this RFC, as its purpose is to illustrate the general design of CKB and avoid too much details.